src/main/webapp/common_util.js

```diff
@@ -13,16 +13,22 @@
 // limitations under the License.
 
 /*
```

Comment:
Can you pass this file through a linter so that these ESLint warnings are removed? Since this file isn't really in conflict w/ other PRs this should mostly be a safe operation.

There's some instructions on how to do that here:
https://eslint.org/docs/user-guide/command-line-interface

Reply:
Trying to run ESLint yields
"ESLint couldn't find the config "google" to extend from. Please check that the name of the config is correct."

Reply:
Let's fix all the linter issues in this PR.